pcsx2/gui/AppConfig.cpp

```diff
@@ -880,6 +879,18 @@ void AppConfig::GSWindowOptions::LoadSave( IniInterface& ini )
 	};
 
 	ini.EnumEntry( L"AspectRatio", AspectRatio, AspectRatioNames, AspectRatio );
+
+	static const wxChar* FMVAspectRatioSwitchNames[] =
+	{
+		// FIXME: fmv_aspect_ratio_switch_labels[] is inverted
+		L"Off", // Standard (4:3)
+		L"4:3", // Widescreen (16:9)
+		L"16:9", // Off
```

Comment:
Why wasn't this fixed before it was merged?

Reply:
Perhaps I should've changed fmv_aspect_ratio_switch_labels instead so the Off option remains at the top instead, will think about it.